base/stats.go

```diff
@@ -431,8 +431,12 @@ type CacheStats struct {
 	RevisionCacheHits *SgwIntStat `json:"rev_cache_hits"`
 	// The total number of revision cache misses.
 	RevisionCacheMisses *SgwIntStat `json:"rev_cache_misses"`
-	// The current length of the pending skipped sequence queue.
+	// The current length of the pending skipped sequence slice.
 	SkippedSeqLen *SgwIntStat `json:"skipped_seq_len"`
+	// The current capacity of the skipped sequence slice
+	SkippedSeqCap *SgwIntStat `json:"skipped_seq_cap"`
+	// The number of sequences currently in the skipped sequence slice
+	NumCurrentSeqsSkipped *SgwIntStat `json:"num_seqs_skipped"`
```

Comment:
I think the naming similarities between the existing `num_skipped_seqs` and `num_seqs_skipped` will just cause problems/confusion.  Maybe `skipped_seq_count`, to make the naming consistent with the other "current state, non-cumulative" stats?